packages/apps/transfer/src/components/Global/AccountText/styles.ts

```diff
@@ -29,12 +30,12 @@ export const StyledAccountContainer = styled('div', {
   gap: '$2',
 });
 
-export const StyledEyeOffIcon = styled(SystemIcons.EyeOffOutline, {
+export const StyledEyeOffIcon = styled(SystemIcon.EyeOffOutline, {
   width: '24px',
   height: '24px',
 });
 
-export const StyledEyeIcon = styled(SystemIcons.EyeOutline, {
+export const StyledEyeIcon = styled(SystemIcon.EyeOutline, {
   width: '24px',
   height: '24px',
 });
```

Comment:
I have a feeling the error might be coming from this. Could you try to use one of the predefined size props on the icon or add more sizes if necessary?

Reply:
I tried things like $16 and $sm but both result in the same error.

Reply:
I meant removing these blocks completely and passing the size as a prop to the icon wherever it's used. The icons already have predefined sizes variants: sm, md, lg, xl, and heroHeader

So wherever it's being used you should be able to apply the 24px size using the `md` prop like this
```
<SystemIcon.EyeOutline size="sm" />
```